variables.tf

```diff
@@ -40,7 +40,8 @@ variable "rds_event_ids" {
 RDS (CloudWatch) Event ID that will trigger the calling of RDS Start Export Task API:
 - Automated snapshots of Aurora RDS: RDS-EVENT-0169
 - Automated snapshots of non-Aurora RDS: RDS-EVENT-0091
-Only automated backups of either RDS Aurora and RDS non-Aurora are supported.
+- Manual snapshots of Aurora/non-Aurora RDS: RDS-EVENT-0075
```

Comment:
That `RDS-EVENT-0075` is okay for Aurora but I think the ID of non-Aurora manual snapshots event is `RDS-EVENT-0042` as specified here: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_Events.Messages.html#USER_Events.Messages.snapshot

Reply:
You're right. I mistook it with the event for clusters which is also listed in that doc. Fixed!